packages/thirdweb/src/wallets/in-app/web/utils/get-enclave-user-status.ts

```diff
@@ -0,0 +1,29 @@
+import type { ThirdwebClient } from "../../../../client/client.js";
+import { getThirdwebBaseUrl } from "../../../../utils/domains.js";
+import { getClientFetch } from "../../../../utils/fetch.js";
+import type { UserStatus } from "../lib/enclave-wallet.js";
+import type { Ecosystem } from "../types.js";
+
+export async function getEnclaveUserStatus(
```

Comment:
can you also move this to /lib/actions? its nice to have all the endpoint wrappers in one place